the-blue-alliance-ios/View Elements/Awards/AwardTableViewCell.swift

```diff
@@ -9,25 +9,40 @@ class AwardTableViewCell: UITableViewCell, Reusable {
         }
     }
     var teamKeySelected: ((_ teamKey: String) -> Void)?
+    private let awardNameLabel = UILabel()
+    private let awardsFlexView = FlexLayoutView()
+    
+    // Mark: - Init
 
-    // MARK: - Reusable
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
```

Comment:
Can we just make this `init` and assume the `style`/`reuseIdentifier` that we want? I can't imagine a situation where we'd want an `AwardTableViewCell` with a different `reuseIdentifier` then the identifier we use for the `Reusable` protocol, or with a different `style`.